samples/jafu-reactive-data-r2dbc/build.gradle.kts

```diff
@@ -1,6 +1,6 @@
 plugins {
 	id("io.spring.dependency-management") version "1.0.11.RELEASE"
-	id("org.springframework.boot") version "2.4.4"
+	id("org.springframework.boot") version "2.5.0"
```

Comment:
2.5.1 already released

Reply:
12 days ago when I submitted this pull request 2.5.1 was not released ;) I am testing it with 2.5.1 and will update my PR very soon